adapters/mqtt-base/src/main/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapter.java

```diff
@@ -1782,38 +1774,26 @@ protected final void onUnsubscribe(final MqttUnsubscribeMessage unsubscribeMsg)
         }
 
         private Future<Void> onCommandSubscriptionRemoved(
```

Comment:
Since there are now cases when a command subscription is removed but this method isn't invoked, I think this should better be renamed (e.g. `closeCommandConsumer`).

Reply:
renamed